packages/oas-resolver/package.json

```diff
@@ -23,7 +23,7 @@
     "oas-kit-common": "^1.0.8",
     "reftools": "^1.1.0",
     "yaml": "^1.8.3",
-    "yargs": "^12.0.5"
+    "yargs": "^13.3.0"
```

Comment:
The sourceclear page you linked to says:

> This issue was fixed in version 13.1.1. However, that version is itself subject to other vulnerabilities, there are no safe versions to upgrade to.

Is that information still current? Will upgrading to 13.3.0 fix all warnings for you?

One of the breaking changes in yargs broke the handling of `require`d positional parameters which is why we're stuck back on `12.x` at the moment. I need to go and see if that was post `13.3.0` or not.

Reply:
Thank you very much for taking on the investigative work, we all greatly appreciate that!